### DISCUSSION TITLE
Border Bug

### DISCUSSION THREAD

**Discussion by GreyFoxGer** [SFX-WoW/Masque#197] [Issues] [answered]
I have found a bug:

https://imgur.com/a/YfCkXT2

How to reproduce:

type /reload, don't open the bags after this. Then kill some mob and loot something. After open the bags and see the border, it will be the same as on gif.

Im sure it related to Masque, because i tried it with Bagnon, Combuctor, tried also different skins, it's all the same bug. So please fix? :)

After reopen the bags, it disappears again... maybe some event are missing?

**Comment by StormFX** [comment]
I just tested with Bagnon and couldn't reproduce it. _Masque_ does not monitor changes by add-ons, so if an add-on is doing something to the icon after it's been skinned and/or not updating it properly, that's the fault of the add-on.

**Comment by GreyFoxGer** [comment]
The problem is definitely here...

i disabled every addon, except bagnon, bagnon facade, and masque with goldpaw skin.

I have have made a video:

https://youtu.be/6QVPxn9xQlw


Test it with goldpaw skin, all another skins have the same problem but they are just hidden it by the border, and you may be overlooked it...

If you using windows 10, you can press win and + key to enable magnifier to see it in detail.

**Comment by StormFX** [comment]
I just tested for 2 hours and was still unable to reproduce the issue. I'm not saying that you're not having it, only that _I'm_ not.

I created a new character, loading only Bagnon, Bagnon_Facade, Masque-2.8.5-Beta-2 and the latest version of Masque_Goldpaw. Note that I had to comment out the "EmptyTexture" portion of the Normal skin due to the empty version covering the icon up . I followed the quest lines, reloading the interface before every kill and then opening the bag only after looting.

I tried with both an empty bag and a nearly full bag, and also tried with the Zoomed skin with the icon clipping disabled. At no point did the icon's attributes change in any way, visibly or in the icons object table.

**Comment by GreyFoxGer** [comment]
Did you tried to switch the items? Try to switch the newly grey item with some green/blue/epic item and see if the problem occurs.

**Comment by GreyFoxGer** [comment]
I just created the new directory where i put the whole wow from scratch.

Still same problem happend... grey items may be not affected, but if i switch the blue item to the place where the grey was, that strange border starting show

**Comment by StormFX** [comment]
Ok, now I see it. It's actually the `IconBorder` that's showing. This seems to be because of an issue with the button metatables. I think Bagnon_Facade is going to need to call an update function for when items are dragged around in the bags.

**Comment by GreyFoxGer** [comment]
Same problem occured with combuctor... how to implement an update function?

**Comment by StormFX** [comment]
Combuctor_Masque and Bagnon_Facade do the exact same thing, since they're both created by the same author and their parent add-ons use the same API (WildPants). You'll need to post an issue on Bagnon_Facade's GitHub.

**Comment by StormFX** [comment, marked as answer]
I posted a ticket and solution on Bagnon_Facade's GitHub.

https://github.com/Jaliborc/Bagnon_Facade/issues/3

**Comment by StormFX** [comment]
Closing this as unrelated.
